core/src/main/kotlin/DokkaBootstrapImpl.kt

```diff
@@ -18,7 +18,13 @@ fun parsePerPackageOptions(arg: String): List<PackageOptions> {
         val reportUndocumented = args.find { it.endsWith("warnUndocumented") }?.startsWith("+") ?: true
         val privateApi = args.find { it.endsWith("privateApi") }?.startsWith("+") ?: false
         val suppress = args.find { it.endsWith("suppress") }?.startsWith("+") ?: false
-        PackageOptionsImpl(prefix, includeNonPublic = privateApi, reportUndocumented = reportUndocumented, skipDeprecated = !deprecated, suppress = suppress)
+        PackageOptionsImpl(
+            prefix = prefix,
+            matchingRegex = null,
```

Comment:
IMHO regex could be a non-nullable String as an empty regex matches only empty Strings

Reply:
I was thinking about making prefix nullable actually. That would allow to validate the user parameters. Something like:

```kotlin
    require(prefix != null || matchingRegex != null) {
         "perPackageOption does not specify prefix neither matchingRegex, it will never be used"
    }
```

Reply:
Damn, I just missed 0.10.1 :-/. Should I rebase this PR somewhere else and make `regex` non-nullable ?